onboarding/views.py

```diff
@@ -382,6 +380,7 @@ def add_user_to_package(self, request, pk=None):
                     package
                 )
 
+
```

Comment:
My `IDE` will complain and describe it as `PEP 8` 'something'.

Reply:
Probably because by PEP standards we should have two paragraphs break between classes and one between methods / functions :) I don't see precise point where your IDE is complaining though.

Reply:
One line in-between methods and within one method is enough

Reply:
The `PEP 8` website you shared me (thx) has about blank lines:
[www.python.org/dev/peps/pep-0008/#blank-lines](https://www.python.org/dev/peps/pep-0008/#blank-lines)

Reply:
Yup, this is what I meant:
"Surround top-level function and class definitions with two blank lines.

Method definitions inside a class are surrounded by a single blank line.

Extra blank lines may be used (sparingly)"
So here for example should be one blank like, not two. I think we could stick to it in future development. :) I will revert other style changes as you suggested (comments etc.).